apps/demo/pages/_app.tsx

```diff
@@ -43,12 +43,12 @@ function MyApp({ Component, pageProps }: AppProps) {
   )
 }
 
-// const client = new LocalClient();
-const client = new Client({
-  clientId: 'some-id',
-  organizationId: 'gctc',
-  branch: 'main',
-})
+const client = new LocalClient()
+// const client = new Client({
+//   clientId: 'some-id',
+//   organizationId: 'gctc',
+//   branch: 'main',
+// })
```

Comment:
I decided to commit this.  I can take it back out, but I feel like we would probably prefer to test locally with `demo`.